src/app/about/page.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
+
+import UnderDevelopment from '@/components/general/UnderDevelopment/UnderDevelopment';
+import { monotonFont } from '@/helpers/font'
+import { BackgroundBeams } from '@/components/ui/background-beams';
+import React from 'react'
+import { motion } from 'framer-motion'
+
+const page = () => {
+  return (
+        <div className="min-h-[100vh] w-full bg-neutral-950 relative flex flex-col items-center justify-center antialiased text-white overflow-x-hidden">
+        <BackgroundBeams />
+        <motion.div
+        initial={{ opacity: 0, y: 30 }}
+        animate={{ opacity: 1, y: 0 }}
+        transition={{ duration: 1, ease: 'easeOut' }}
+        className="relative flex flex-col items-center z-10"
+            >
+        <span
+          className={`h-20 md:h-30 lg:h-35 text-7xl md:text-8xl lg:text-9xl bg-clip-text text-transparent bg-gradient-to-r from-purple-500 via-pink-500 to-yellow-500 text-center logo-gradient-shift-animation ${monotonFont.className}`}
+        >
+          ABOUT US
+        </span>
+        <div className="mt-10 flex justify-center w-full">
+          <UnderDevelopment />
+        </div>
```

Comment:
We need to wrap this JSX into feature-flag conditionals.


```suggestion
   {AboutPage ?? (<></>) : 
   ( <div className="mt-10 flex justify-center w-full">
          <UnderDevelopment />
    </div>
    )
    }

```

Reply:
Ok I will do it asap

Reply:
Kindly, make sure to update the feature-flags in the `useFeatureFlag.ts` hook and use it accordingly.